src/components/Dashboard/DashboardTile/DashboardTile.js

```diff
@@ -133,7 +133,13 @@ export default class DashboardTile extends React.Component {
   componentDidUpdate = (prevProps) => {
     // If query or title change from props (due to undo for example), update state
     if (_get(this.props, 'tile.query') !== _get(prevProps, 'tile.query')) {
-      this.setState({ query: _get(this.props, 'tile.query') })
+      this.setState({ query: _get(this.props, 'tile.query') }, () => {
```

Comment:
https://chataai.atlassian.net/browse/ANT-584
https://chataai.atlassian.net/browse/ANT-592
make sure the supportedDispalyTypes update accordingly when query changes